### PR TITLE
Implement Mobify Analytics

### LINKED ISSUE #9209: Intent to Implement Mobify Analytics as vendor to amp-analytics

### PR DESCRIPTION
**Pull request by kschmidtdev**
- Adds support for Mobify Analytics via tracking pixel

Closes #9209

### CODE REVIEW

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -610,6 +610,18 @@
 </amp-analytics>
 <!-- End Mediator.media example -->
 
+<!-- Mobify tracking -->
```
Comment (zhouyx) [inline examples/analytics-vendors.amp.html]: Please add your name to this [form](https://github.com/ampproject/amphtml/blob/master/examples/analytics-vendors.amp.html#L45) as well for us to find your test more convienient. Thanks.

Comment (kschmidtdev) [reply examples/analytics-vendors.amp.html]: Done 👍

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -1100,6 +1100,63 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
...
+      'pageload': {
+        'on': 'visible',
+        'request': 'pageload',
+      },
+      'pagedcl': {
+        'on': 'visible',
+        'request': 'pagedcl',
+      },
+    },
+    'transport': {
```
[the remark this answers is not in the document]
Comment (kschmidtdev) [reply extensions/amp-analytics/0.1/vendors.js]: I can confirm that now we support `beacon` and `image` 😄

**Comment by zhouyx** [comment]
@kschmidtdev sorry about the delay here. Everything looks good! In order to merge you'll need to sync with master to fix the percy visual diff test.
Please let me know.

**Comment by kschmidtdev** [comment]
@zhouyx No problem and thank you! I have merged the latest changes from master in.

**Comment by zhouyx** [comment]
@kschmidtdev Thanks! PR merged!

**Pull request merged**